core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -243,7 +243,8 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
-  test("add and list jar files") {
+  ignore("SPARK-51318: Remove `jar` files from Apache Spark repository and disable affected " +
```

Comment:
Can we create the jar during test runtime instead of simply disabling it?

Reply:
The agreement in the e-mail thread on dev@spark.apache.org I referred to is to disable affected tests. As discussed in the same e-mail thread, there are several ways to provide jars at runtime and building them is one of the option. It is outside of the scope for this PR.

Reply:
I don't think we reach the agreement there, and we did not also go for a vote. I disagree with disabling the tests.

Reply:
It was voted on the e-mail thread I posted and there were no -1 votes.

Reply:
Do you please mind sharing the link of vote? I cannot see in the dev mailing list in Spark.

Reply:
There is no any other thread except the one I mentioned. I don't see why you expect a separate voting thread. In many cases votes are casted on the discussion thread. For example you [casted vote](https://lists.apache.org/thread/frrxhvbqtp6594k1tvs1wwmd4b2zg8h3) on the [discussion thread](https://lists.apache.org/thread/2sbdknnfn119msdq52dq5wk90jy0hc88).